Reyes_JohnRuiz/countries.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Countries of the World</title>
+    <link rel="stylesheet" href="./countries/countries.css" />
+  </head>
+  <body>
+    <div class="container">
+      <h1 class="centered">Countries of the World</h1>
+      <input
+        type="text"
+        id="countryInput"
+        placeholder="Enter country name..."
+      />
+      <button onclick="searchCountry()">Search</button>
+      <div class="country-details centered" id="countryDetails"></div>
+      <div class="same-region-countries" id="sameRegionCountries"></div>
+    </div>
```

Comment:
IDs should be written in snake case. All lower case separated by underscore.
Also, don't separate the closing tag in a new line. 
```suggestion
      <input
        type="text"
        id="country_input"
        placeholder="Enter country name..." />
      <button onclick="searchCountry()">Search</button>
      <div class="country-details centered" id="country_details"></div>
      <div class="same-region-countries" id="same_region_countries"></div>
    </div>
```
See the reference for clarifications
https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit?fbclid=IwAR2_UbyCcqMcmiNZWfIfPBh3nnnFc3lCbZ7FOtRkNSo6-6gv0e-dbYmswHs#slide=id.g24da8de92d7_0_5